adlfs/lib.py

```diff
@@ -198,6 +228,7 @@ def call(self, op, path='', **kwargs):
         """
         if op not in self.ends:
             raise ValueError("No such op: %s", op)
+        self._check_token()
```

Comment:
so this means we will check the token on every request, right? There is no scenario where we could inadvertently end up executing prior to checking the token and end up with an expired token prior to refreshing, right?